core/cap-0037.md

```diff
@@ -314,35 +314,35 @@ liquidity pool defined as `LiquidityPoolEntry`.
   balances do not exist, `DEPOSIT_POOL_LIQUIDITY_INSUFFICIENT_AMOUNT` error
   returned.
 - Deposited assets ratio calculated as  
-  ![`r=a/b`](https://render.githubusercontent.com/render/math?math=p%3D%5CLarge%5Cfrac%7Ba%7D%7Bb%7D)  
+  $r=\Large\frac{a}{b}$
   Current pool price:  
-  ![`P=A/B`](https://render.githubusercontent.com/render/math?math=P%3D%5CLarge%5Cfrac%7BA%7D%7BB%7D)  
+  $P=\Large\frac{A}{B}$  
   where:
     - ***A*** and ***B*** - amounts of asset A and asset B currently in the pool
     - ***a*** and ***b*** - `maxAmountA` and `maxAmountB` defined in the
       operation
 - Maximum allowed price deviation is controlled by the `priceAccuracy` (***d***)
   parameter.  
-  ![`|(1-r/P)|<=1/d`](https://render.githubusercontent.com/render/math?math=%5Cleft%5Cvert%201-%5CLarge%5Cfrac%7Br%7D%7BP%7D%5Cright%5Cvert%5Cleq%5CLarge%5Cfrac%7B1%7D%7Bd%7D)  
+  $\left\vert 1-\Large\frac{r}{P}\right\vert\leq\Large\frac{1}{d}$  
   In case of a significant deviation, `DEPOSIT_POOL_LIQUIDITY_PRICE_MISMATCH`
   error returned. This check is ignored if the pool doesn't exist
   or `priceAccuracy` equals to zero.
 - Effective token amounts that can be deposited are adjusted to follow the
   current price.  
   If
-  ![`a*B > b*A`](https://render.githubusercontent.com/render/math?math=a%20%5Ccdot%20B%20%3E%20b%20%5Ccdot%20A)
+  $a\cdot B > b\cdot A$
   , token A deposit amount is adjusted as  
-  ![`a=ceil(bd*P)`](https://render.githubusercontent.com/render/math?math=a%3Dceil(b_d%20%5Ccdot%20P))  
+  $a=ceil(b_d\cdot P)$  
   otherwise  
-  ![`b=ceil(ad/P)`](https://render.githubusercontent.com/render/math?math=b%3Dceil%5CLarge(%5Cfrac%7Ba_d%7D%7BP%7D))  
+  $b=ceil\Large(\frac{a_d}{P})$  
   where:
-    - ![`ad=min(maxAmountA,accountBalanceA)`](https://render.githubusercontent.com/render/math?math=a_d%3Dmin(maxAmountA%2CaccountBalanceA))
-    - ![`bd=min(maxAmountB,accountBalanceB)`](https://render.githubusercontent.com/render/math?math=b_d%3Dmin(maxAmountB%2CaccountBalanceB))
+    - $a_d=min(maxAmountA,accountBalanceA)$
+    - $b_d=min(maxAmountB,accountBalanceB)$
     - ***a*** and ***b*** – maximum effective amounts of tokens A and B that can
       be deposited to the pool. If the actual deposited amount of any token
       equals zero, `DEPOSIT_POOL_LIQUIDITY_INSUFFICIENT_AMOUNT` error returned.
 - Stake weight calculated as  
-  ![`s=floor(S*√((a*b)/(A*B)))`](https://render.githubusercontent.com/render/math?math=s%3Dfloor(S%5Ccdot%5CLarge%5Csqrt%7B%5Cfrac%7Ba%5Ccdot%20b%7D%7BA%5Ccdot%20B%7D%7D%5Cscriptsize))  
+  $s=floor(S\cdot\sqrt{\Large\frac{a\cdot b}{A\cdot B}})$  
```

Comment:
Seems like a good opportunity to use `\left\lfloor{...}\right\rfloor` and same for `ceil` since we're here